src/users/users.service.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@nestjs/common';
+import { CreateUserDto } from './dto/create-user.dto';
+import { UpdateUserDto } from './dto/update-user.dto';
+import { UsersRepository } from './users.repository';
+
+@Injectable()
+export class UsersService {
+  constructor(private readonly usersRepository: UsersRepository) {}
+
+  create(createUserDto: CreateUserDto) {
```

Comment:
Please add try-catch construction to every method. Also add all-exception-filter to catch all errors and set it on app.module.ts.